swarm/network/kademlia/kademlia.go

```diff
@@ -426,3 +444,15 @@ func (self *Kademlia) String() string {
 	rows = append(rows, "=========================================================================")
 	return strings.Join(rows, "\n")
 }
+
+//We have to build up the array of counters for each index
+func (self *Kademlia) initMetricsVariables() {
```

Comment:
@lmars please review this func, added as for your own review comments. Want to make sure I am correctly initializing stuff based on kad table size and indexes. Thanks!